src/modules/getScores.js

```diff
@@ -6,8 +6,12 @@ const getScores = async () => {
   let leaders = '';
   scores.sort((a, b) => b.score - a.score);
   scores.forEach((score, i) => {
+    let odd = 'odd';
+    if (i % 2 === 0) {
+      odd = 'even';
+    } else { odd = 'odd'; }
```

Comment:
- [ OPTIONAL ] Great working on the styles and thw wireframe but I noticed that the functionality isn't working as expected so please fix the issue to let your project work properly 😊🙏🏻💯💯🔥🔥✅✅

Reply:
@Reem-lab Alright thank you very much dear reviewer